openwisp_radius/settings.py

```diff
@@ -113,6 +113,12 @@ def get_default_password_reset_url(urls):
         "location": "disabled",
     },
 )
+ADDITIONAL_RADIUS_CHECKS = []
```

Comment:
Did you mean to allow overriding this? It looks unoverridable, I think we can remove this and anything related to it to reduce the new lines of code added.